Makefile

```diff
@@ -74,6 +74,7 @@ INSTALL_PYLIB_FILES = \
     _stbt/imgproc_cache.py \
     _stbt/imgutils.py \
     _stbt/irnetbox.py \
+    _stbt/libstbt-30.so \
```

Comment:
It isn't a public API (we don't expect others to link to it) so we might as well call it `libstbt.so` and save the hassle of bumping the version number every release.

Reply:
Fixed.